lib/ansible/module_utils/basic.py

```diff
@@ -179,6 +167,11 @@
 from ansible.module_utils.common._utils import get_all_subclasses as _get_all_subclasses
 from ansible.module_utils.parsing.convert_bool import BOOLEANS, BOOLEANS_FALSE, BOOLEANS_TRUE, boolean
 
+try:
+    from ansible.module_utils.common._json_compat import json
+except ImportError as e:
+    print('\n{{"msg": "Error: ansible requires the stdlib json {0}", "failed": true}}'.format(to_native(e)))
+    sys.exit(1)
```

Comment:
This import should likely be moved back where the original code was, so that it trips early, and other places that import `_json_compat` don't cause pre-mature exceptions that aren't handled

Reply:
Ok. It needs to be after `to_native()` which is why I put it down here, to avoid having to move that import. But I'll do the needful.